src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -59,15 +59,17 @@ emit('toggle-navigation', {
 		role="navigation"
 		:class="{'app-navigation--close':!open }">
 		<NcAppNavigationToggle :open="open" @update:open="toggleNavigation" />
-		<slot />
 
-		<!-- List for Navigation li-items -->
-		<ul class="app-navigation__list">
-			<slot name="list" />
-		</ul>
+		<div :aria-hidden="ariaHidden" class="app-navigation__content">
+			<slot />
+			<!-- List for Navigation li-items -->
+			<ul ref="navigationList" class="app-navigation__list">
```

Comment:
The ref is not used, no?
```suggestion
			<ul class="app-navigation__list">
```